src/internal/client.ts

```diff
@@ -1626,4 +1624,108 @@ export class TypedClient {
 
     await this.makeRequestAsyncOmit({ method, bucketName, query, headers }, payload)
   }
+
+  async setTagging(taggingParams: PutTaggingParams): Promise<void> {
```

Comment:
Should add `private` identifier to make sure this function is never called from outside of this class.

```suggestion
  private async setTagging(taggingParams: PutTaggingParams): Promise<void> {
```